File: 문제풀이/chap2-5/prob2-5-3.cpp
#include <iostream>
using namespace std;

int main() {
	long long N;
	long long Answer = 1;
	cin >> N;
	for (int i = 2; i <= N; i++) Answer *= i; // Answer에 i를 곱함
	cout << Answer << endl;
	return 0;
}

File: 문제풀이/chap2-5/prob2-5-3.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		// 입력
		long N = sc.nextInt();

		// 답 계산
		long Answer = 1;
		for (int i = 1; i <= N; i++) Answer *= i;

		// 출력
		System.out.println(Answer);
	}
};